docs/source/about.rst

```diff
@@ -1,3 +1,5 @@
+.. warning::
+    This page was developed by a community member in 2015 and is not being actively maintained.  Please refer to chapel-lang.org/docs instead.
```

Comment:
Maybe we should drop "community member" and say something like "This page was last updated in 2015 and is not being....."

Reply:
I'd be fine with that change personally. I'll give @bradcray a chance to weigh in before pursuing the change, since he proposed the original text here.

Reply:
I'd be fine with that.  Or we could just make it slightly vaguer, like "was contributed in 2015"

Reply:
Also fine with that. Sound good to you @e-kayrakli ?

(Clearly this is a task that requires the experience of 2 CS PhDs)

Reply:
:)

That sounds fine to me, too